### PR TITLE
Form Submission

### PR DESCRIPTION
**Pull request by arranfw**
This allows us to submit partial form values to the API

**NOTE** You'll either need to run the webapp through docker or add `NEXT_PUBLIC_API_URL=http://localhost:4000/api/v1` to a file called `.env.local` in `apps/web`

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -51,6 +51,7 @@ services:
     ports:
       - '3000:3000'
     environment:
+      - NEXT_PUBLIC_API_URL=http://localhost:4000/api/v1
```
Comment (dbayly-freshworks) [inline docker-compose.yml:54]: Will this work when deployed? Should we create an environment variable?

Comment (arranfw) [reply docker-compose.yml:54]: oh yeah this is going to need to be added to TF, thanks

Comment (hitankar-freshworks) [reply docker-compose.yml:54]: This will be dynamic based on what cloudfront sets. I think LTC has a way to set this based on the environment.

**Review by hitankar-freshworks — APPROVED** [review]
One change is requested. Dynamically set url in the Makefile and reference that in `docker-compose.yml`.

**Pull request merged**